src/MSBuild/app.amd64.config

```diff
@@ -49,7 +49,7 @@
 
         <dependentAssembly>
           <assemblyIdentity name="Microsoft.IO.Redist" culture="neutral" publicKeyToken="cc7b13ffcd2ddd51" />
-          <bindingRedirect oldVersion="0.0.0.0-99.9.9.9" newVersion="6.0.0.0" />
+          <bindingRedirect oldVersion="0.0.0.0-6.0.0.1" newVersion="6.0.0.0" />
```

Comment:
There seems like the version was not reverted as it was before.

Reply:
`newVersion` is the version which image loader picks. `oldVersion` represents a range for assembly that is requested. I believe it's still valid.